db/migrate/20180821112856_create_openshift_service_catalog_tables.rb

```diff
@@ -0,0 +1,114 @@
+class CreateOpenshiftServiceCatalogTables < ActiveRecord::Migration[5.0]
+  def change
+    create_table :container_service_brokers, :id => :bigserial, :force => :cascade do |t|
+      t.string :name
+      t.string :kind
+      t.string :ems_ref
+      t.integer :resource_version
+      t.text :url
+
+      t.references :ems_id, :type => :bigint, :index => true, :references => :ext_management_systems
+      t.references :container_projects, :type => :bigint, :index => true
+
+      t.jsonb :extra
+
+      t.datetime :ems_created_on
+      t.timestamps
+    end
+
+    create_table :container_service_classes, :id => :bigserial, :force => :cascade do |t|
+      t.string :name
+      t.string :kind
+      t.string :status
+      t.string :ems_ref
+      t.integer :resource_version
+      t.text :description
+      t.boolean :bindable
+      t.boolean :plan_updatable
+
+      t.references :ems_id, :type => :bigint, :index => true, :references => :ext_management_systems
+      t.references :container_projects, :type => :bigint, :index => true
+      t.references :container_service_brokers, :type => :bigint, :index => true
+
+      t.jsonb :extra
+
+      t.datetime :ems_created_on
+      t.timestamps
+    end
+
+    create_table :container_service_plans, :id => :bigserial, :force => :cascade do |t|
+      t.string :name
+      t.string :kind
+      t.string :status
+      t.string :ems_ref
+      t.integer :resource_version
+      t.text :description
+      t.boolean :free
+
+      t.references :ems_id, :type => :bigint, :index => true, :references => :ext_management_systems
+      t.references :container_projects, :type => :bigint, :index => true
+      t.references :container_service_classes, :type => :bigint, :index => true
+
+      t.jsonb :extra
+
+      t.datetime :ems_created_on
+      t.timestamps
+    end
+
+    create_table :container_service_instances, :id => :bigserial, :force => :cascade do |t|
+      t.string :name
+      t.string :kind
+      t.string :status
+      t.string :ems_ref
+      t.string :secret_name
+      t.integer :resource_version
+
+      t.references :ems_id, :type => :bigint, :index => true, :references => :ext_management_systems
+      t.references :container_projects, :type => :bigint, :index => true
+      t.references :container_service_classes, :type => :bigint, :index => {:name => 'csi_on_container_service_classes'}
+      t.references :container_service_plans, :type => :bigint, :index => true
+
+      t.jsonb :parameters
+      t.jsonb :parameters_from
+      t.jsonb :extra
+
+      t.datetime :ems_created_on
+      t.timestamps
+    end
+
+    create_table :container_service_bindings, :id => :bigserial, :force => :cascade do |t|
```

Comment:
According to the documentation included in this PR, a service binding represents a binding of a service instance to an application. Although the doc has a TODO to define what an application is, do we need this if we don't have a table for container_applications?

Reply:
this mentions application pods https://github.com/kubernetes-incubator/service-catalog/blob/master/docs/resources.md#servicebinding

right now, I am not sure what it is exactly :-)

Reply:
@Ladas @gtanzillo - The binding was a way ( via environment variables ) to provision a multiple service instances all of which are needed to form a single service.  E.g. MediaWiki: provision the wiki and the db so when the service instance is provisioned it includes both the frontend and the db as a single provisioned entity.

Reply:
Ok, so this is something we need?

Or we don't, because you've used past tense for 'The binding was a way' :-)